src/App.js

```diff
@@ -13,6 +14,8 @@ function App() {
      <Routes>
      <Route path="/" element={<Nav/>}/>
      <Route path="/reviews" element={<Reviews/>}/>
+     <Route path="/review/:review_id" element={<SingleReview/>}/>
+     {/* <Route path="/review/:review_id/comments" element={<CommentCard/>}></Route> */}
```

Comment:
Please remove any commented out code. I dont think you'd really need a route just for a comment anyway, so should be good to be taken out